docs/docs/03-main-concepts.md

```diff
@@ -0,0 +1,79 @@
+## Main Concepts
+
+
+### Unbundled Development
+
+![webpack vs. snowpack diagram](/img/snowpack-unbundled-example-3.png)
+
+**Unbundled development** is the idea of shipping individual files to the browser during development. Files can still be built with your favorite tools (like Babel, TypeScript, Sass) and then loaded individually in the browser with dependencies thanks to ESM `import` and `export` syntax. Any time you change a file, Snowpack only ever needs to rebuild that single file.
+
+The alternative is **bundled development.** Almost every popular JavaScript build tool today focuses on bundled development. Running your entire application through a bundler introduces additional work and complexity to your dev workflow that is unnecessary now that ESM is widely supported. Every change -- on every save -- must be rebundled with the rest of your application before your changes can be reflected in your browser. 
+
+Unbundled development has several advantages over the traditional bundled development approach:
+
+- Builds are faster.
+- Builds are predictable.
+- Builds are easy to reason about & debug: `build(input) -> output`
```

Comment:
```suggestion
- Builds are easy to reason about & debug because 1 input file == 1 output file (i.e. `build(input) -> output`)
```

I think the function notation is really cool but I think it requires explanation. Maybe something like this?

Reply:
Hmm, okay I'll remove it from this line to keep it simple/tight, and maybe give it its own space below.